editoast/src/models/train_schedule.rs

```diff
@@ -107,14 +107,32 @@ impl From<TrainScheduleChangeset> for TrainSchedule {
     }
 }
 
+/// Returns the timetable with the main information about a train
 #[derive(Serialize, Debug, PartialEq, Clone, Queryable)]
-pub struct TrainScheduleDetails {
+pub struct LightTrainScheduleDetails {
     pub id: i64,
     pub train_name: String,
     pub departure_time: f64,
     pub train_path: i64,
 }
 
+#[derive(Serialize, Debug, PartialEq, Clone, Default)]
+pub struct MechanicalEnergyConsumedBaseEco {
+    pub base: f64,
+    pub eco: Option<f64>,
+}
+
+/// Returns the timetable with the train's important route information (distance travelled, arrival time, etc.)
+#[derive(Serialize, Debug, PartialEq, Clone, Queryable)]
+pub struct DetailedTrainSchedule {
```

Comment:
TrainScheduleDetails wasn't a good name, but I think this is worse. It's not detailed, it's a summary (a real TrainSchedule has *more details*)

The timetable is detailed, not the train schedule.

Reply:
Is `TrainScheduleSummary` better?

Reply:
You got my drift ;)